src/components/ebay-combobox/component.js

```diff
@@ -15,11 +15,19 @@ export default {
     },
 
     isExpanded() {
-        return this.expander.expanded;
+        return this.expander && this.expander.expanded;
```

Comment:
Not sure what you mean. It does not have any collateral effects on state

Reply:
If you mean change it to `get isExpanded(){ ...}`.  The use of a function makes things more explicit. 

Getter have a place but should be used everywhere. My personal feeling about getters and setters is that they make code harder to read because they look exactly like normal properly access but do something more complicated. Code is hard to read when you expect it to do one thing but it does another.

You can also read more opinions here: https://www.quora.com/Why-are-getters-and-setters-increasingly-considered-an-anti-pattern-in-programming-circles